src/components/Dialog/Dialog.jsx

```diff
@@ -0,0 +1,97 @@
+import _ from 'lodash';
+import React from 'react';
+import Overlay from '../Overlay/Overlay';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createLucidComponentDefinition }  from '../../util/component-definition';
+
+const boundClassNames = lucidClassNames.bind('&-Dialog');
+
```

Comment:
There's one place we shouldn't do this for documentation sake. Because our prop types are extracted from the AST, it can't read the value of the constant, so people would see `SMALL` in the docs. I'm going to leave it as strings in the propTypes but use the constants elsewhere.